dynamic_programming/maximum-sum-increasing-subsequence.py

```diff
@@ -0,0 +1,32 @@
+#############################
+# Author: Ujjal Das( github.com/ujjaldas132)
+# File: lis.py
+# comments:Given an array A . Fin d the sum of maximum sum increasing subsequence of the given array.
+#           if A=[1 101 2 3 100 4 5] then answer shold be 106>>>>>>1,2,3,100
+#############################
+
+def max_sum(a):
+    n=len(arr)
+    
```

Comment:
In the automated tests __flake8__ is complaining because the paramenter is __a__ but we are taking the length of __arr__.  Let's pick one or the other.  (I find __arr__ to be slightly more intuitive)  or even __array_of_integers__.